x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/SqlTestUtils.java

```diff
@@ -81,4 +86,40 @@ public static Literal literal(Source source, Object value) {
         }
         return new Literal(source, value, SqlDataTypes.fromJava(value));
     }
+
+    public static String randomOrderByAndLimit(int noOfSelectArgs, Random rnd) {
+        StringBuilder sb = new StringBuilder();
+        if (randomBoolean()) {
+            sb.append(" ORDER BY ");
+
+            List<Integer> shuffledArgIndices = IntStream.range(1, noOfSelectArgs + 1).boxed().collect(Collectors.toList());
+            Collections.shuffle(shuffledArgIndices, rnd);
+            for (int i = 0; i < noOfSelectArgs; i++) {
+                sb.append(shuffledArgIndices.get(i));
+                switch (randomInt(2)) {
```

Comment:
Or you could have used a `randomBoolean()`? If `true` then `DESC`, if `false` then `ASC`.

Reply:
I have 3 values so that it skips both -> implied ASC.